tinnsleep/signal.py

```diff
@@ -19,4 +20,4 @@ def rms(epochs):
         Root Mean Square Amplitude
     """
 
-    return np.sqrt(np.mean(epochs ** 2, axis=1))
+    return np.sqrt(np.mean(epochs ** 2, axis=axis))
```

Comment:
great !

Don't forget to skip exactly ONE blank line at the end of each .py file (PEP8)